201-zerto-zca-vpn/azuredeploy.json

```diff
@@ -0,0 +1,398 @@
+         }
+      },
+      "subnetName":{  
+         "type":"string",
+         "defaultValue":"Subnet1",
+         "metadata":{  
+            "description":"Name for the Azure Subnet"
+         }
+      },
+      "subnetPrefix":{  
+         "type":"string",
+         "metadata":{  
+            "description":"CIDR block for VM subnet, subset of azureVNetAddressPrefix address space"
+         }
+      },
+      "gatewaySubnetPrefix":{  
+         "type":"string",
+         "defaultValue":"10.3.200.0/29",
+         "metadata":{  
+            "description":"CIDR block for gateway subnet, subset of azureVNetAddressPrefix address space"
+         }
+      },
+      "gatewayPublicIPName":{  
+         "type":"string",
+         "defaultValue":"azureGatewayIP",
+         "metadata":{  
+            "description":"Name for public IP resource used for the new azure gateway"
+         }
+      },
+      "gatewayName":{  
+         "type":"string",
+         "defaultValue":"azureGateway",
+         "metadata":{  
+            "description":"Name for the new gateway"
+         }
+      },
+      "gatewaySku":{  
+         "type":"string",
+         "defaultValue":"Basic",
+         "allowedValues":[  
+            "Basic",
+            "Standard",
+            "HighPerformance"
+         ],
+         "metadata":{  
+            "description":"The Sku of the Gateway. This must be one of Basic, Standard or HighPerformance."
+         }
+      },
+      "connectionName":{  
+         "type":"string",
+         "defaultValue":"Azure2Other",
+         "metadata":{  
+            "description":"Name for the new connection between Azure VNet and other network"
+         }
+      },
+      "sharedKey":{  
+         "type":"string",
+         "metadata":{  
+            "description":"Shared key (PSK) for IPSec tunnel"
+         }
+      },
+      "vmName":{  
+         "type":"string",
+         "defaultValue":"zca",
+         "metadata":{  
+            "description":"Name of the Zerto VM to create"
+         }
+      },
+      "adminUsername":{  
+         "type":"string",
+         "metadata":{  
+            "description":"Username for the VM"
+         }
+      },
+      "adminPassword":{  
+         "type":"securestring",
+         "metadata":{  
+            "description":"User password for the VM"
+         }
+      },
+      "storageAccountType":{  
+         "type":"string",
+         "defaultValue":"Standard_LRS",
+         "allowedValues":[  
+            "Standard_LRS",
+            "Standard_ZRS",
+            "Standard_GRS",
+            "Standard_RAGRS",
+            "Premium_LRS"
+         ],
+         "metadata":{  
+            "description":"Storage Account Type"
+         }
+      }
+   },
+   "variables":{  
+      "vnetID":"[resourceId('Microsoft.Network/virtualNetworks', parameters('virtualNetworkName'))]",
+      "gatewaySubnetRef":"[concat(variables('vnetID'),'/subnets/','GatewaySubnet')]",
```

Comment:
just use resourceID and remove the vnetID above

Reply:
done